sim/paladin/inquisition.go

```diff
@@ -0,0 +1,73 @@
+package paladin
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (paladin *Paladin) registerInquisition() {
+	if paladin.Talents.InquiryOfFaith == 0 {
+		return
+	}
+
+	actionId := core.ActionID{SpellID: 84963}
+	hpMetrics := paladin.NewHolyPowerMetrics(actionId)
+	inquisitionDuration := 4 * time.Second * time.Duration([]float64{0, 1.66, 2.33, 3.0}[paladin.Talents.InquiryOfFaith])
```

Comment:
when converting from float to duration you lose any float precision. Change this to milliseconds to keep the .66 and .33 fractions